src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/ThemeControllerTests.cs

```diff
@@ -19,79 +19,68 @@ public class ThemeControllerTests
         [Fact]
         public async Task GetThemes()
         {
-            var fileStorageService = new Mock<IFileStorageService>(MockBehavior.Strict);
-            var methodologyService = new Mock<IMethodologyService>(MockBehavior.Strict);
+            var publicationService = new Mock<IPublicationService>(MockBehavior.Strict);
 
-            fileStorageService
-                .Setup(
-                    s => s.GetDeserialized<IEnumerable<ThemeTree<PublicationTreeNode>>>(
-                        "publications/tree.json"
-                    )
-                )
+            publicationService
+                .Setup(s => s.GetPublicationTree())
                 .ReturnsAsync(
                     new List<ThemeTree<PublicationTreeNode>>
                     {
-                        new ThemeTree<PublicationTreeNode>
+                        new()
                         {
                             Topics = new List<TopicTree<PublicationTreeNode>>
                             {
-                                new TopicTree<PublicationTreeNode>
+                                new()
                                 {
                                     Publications = new List<PublicationTreeNode>
                                     {
-                                        new PublicationTreeNode()
+                                        new()
                                     }
                                 }
                             }
                         }
                     }
                 );
 
-            var controller = BuildThemeController(fileStorageService.Object,
-                methodologyService.Object);
+            var controller = BuildThemeController(publicationService.Object);
 
             var result = await controller.GetThemes();
 
-            Assert.Single(result.Value);
+            Assert.Single(result);
```

Comment:
Made this change but had to use the null-forgiving operator for any subsequent calls to avoid compiler warnings e.g. `theme!.topics`. 

Think this is mostly an issue with xUnit as I don't _think_ it has proper nullability annotations currently. This means methods like `Assert.Single` don't  play nicely when we have nullable enabled. 

Using `Assert.NotNull` doesn't help either :upside_down_face: The only other option is to use `Debug.Assert(theThing is not null);` instead, but this kinda sucks as xUnit asserts should be able to do this for us...

I'd suggest we don't need to be too fussy about assigning variables to `Assert.Single` in other cases.

Reply:
Bah that's a shame - I wonder if there are updates in the future planned around this.  Thanks for the info